src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -1377,7 +1381,7 @@ private static bool ValidatePropertyExistence(BlittableJsonReaderObject cmd, str
             return true;
         }
 
-        private void SetValueForTypedDatabaseCommand(ClusterOperationContext context, string type, BlittableJsonReaderObject cmd, long index, Leader leader, out object result)
+        private UpdateValueForDatabaseCommand SetValueForTypedDatabaseCommand(ClusterOperationContext context, string type, BlittableJsonReaderObject cmd, long index, Leader leader, out object result, bool notifyDatabase = true)
```

Comment:
where is this used?

Reply:
All that was done within the `UpdateBackupHistory` method was moved to the `UpdatePeriodicBackupStatusCommand.Execute` method. Now we notify only one time, as it always was.